internal/model/packet.go

```diff
@@ -333,6 +365,6 @@ func (p *Packet) Log(logger Logger, direction int) {
 		p.ACKs,
 		p.LocalSessionID,
 		p.RemoteSessionID,
-		len(p.Payload),
+		payloadLen,
```

Comment:
len(p.Payload) == 0 for p.Payload = nil, so I am not sure we need this change.

Reply:
you're right - I think this morphed from some other logging that was choking on nil. fixed.